templates/interstitial.html

```diff
@@ -1,11 +1,12 @@
 {% extends 'layouts/_questionnaire.html' %}
+{% from 'macros/helpers.html' import format_paragraphs %}
 
 {% set save_on_signout = true %}
 
 {% set continue_button_text = _("Continue") %}
 
 {% block form_content %}
-  {% set interstitial_instruction = format_paragraphs(block.content.instruction) if block.content.instruction else None %}
+  {% set interstitial_instruction = format_paragraphs([block.content.instruction]) if block.content.instruction else None %}
```

Comment:
We should be able to pass an empty iterable to `format_paragraph` and have it not output anything by using the `{% else %}` operator when we iterate
```suggestion
  {% set interstitial_instruction = format_paragraphs([block.content.instruction])
```